MultiSlider.js

```diff
@@ -240,20 +243,27 @@ class MultiSlider extends Component {
     } = this.state;
 
     if (this.state.disabled) {
-      console.log('------------------00001');
       return;
     }
-    console.log('------------------111111111');
-
 
     let nextX = this._getValue(gestureState.dx);
 
-    if (currentThumb === 'left' && (rightValue._value - nextX) > this.props.minSpace) {
-      leftValue.setValue(nextX);
-      this._fireChangeEvent('onLeftValueChange');
-    } else if (currentThumb === 'right' && (nextX - leftValue._value) > this.props.minSpace) {
-      rightValue.setValue(nextX);
-      this._fireChangeEvent('onRightValueChange');
+    if(this.props.allowSameValues) {
```

Comment:
Hi huszy. Thank you for contribution!

I have reviewed the code. But I don't think it's necessary to use so big `if` block. Do you mind use another cleaner way to achieve it? 
For example.
 
```javaScript
 let space = this.props.allowSameValues ? 0 : this.props.minSpace;
...
if (currentThumb === 'left' && (rightValue._value - nextX) > space) {...}
```